.github/workflows/git-artifacts.yml

```diff
@@ -84,6 +84,7 @@ jobs:
           test -n "$ARTIFACTS_TO_BUILD" || {
             ARTIFACTS_TO_BUILD="installer portable archive mingit"
             test "$ARCHITECTURE" = aarch64 || ARTIFACTS_TO_BUILD="$ARTIFACTS_TO_BUILD mingit-busybox"
+            test "$ARCHITECTURE" != x86_64 || ARTIFACTS_TO_BUILD="$ARTIFACTS_TO_BUILD nuget"
```

Comment:
This is a bit of a weird double negative. If the architecture is not not `x86_64`,  include `nuget` in the artifacts to build. It might make sense to express this and the `aarch64` test like 
`test "$ARCHITECTURE" = x86_64 &&`

Reply:
Ah, `set -e` explains it.